types/hsl-to-rgb-for-reals/hsl-to-rgb-for-reals-tests.ts

```diff
@@ -0,0 +1,3 @@
+import hslToRgb = require('hsl-to-rgb-for-reals');
+
+hslToRgb(223, 0.44, 0.56); // $ExpectType [number, number, number]
```

Comment:
```suggestion
hslToRgb(223, 0.44, 0.56); // $ExpectType [red: number, green: number, blue: number]
```
(if you accept named tuple change)